pkgs/applications/editors/emacs/elisp-packages/manual-packages/codeium/default.nix

```diff
@@ -0,0 +1,28 @@
+{ trivialBuild, fetchFromGitHub, pkgs, lib, }:
+trivialBuild {
```

Comment:
Please use `melpBuild` or `elpaBuild`.

Packages built by `trivialBuild` will not be activated[1][2] by Emacs's builtin package manager `package.el` when Emacs starts.  So some issues occur:
- autoloaded commands does not work out-of-the-box: you have to call `(require 'foo-autoloads)` first before calling an autoloaded command
- info manual does not show in `C-h i`

Maybe this issue of `trivialBuild` should be documented somewhere? @adisbladis @AndersonTorres 

[1]: its package dir is [not searched](https://github.com/emacs-mirror/emacs/blob/310ed338c195c755b11e0c62bde9629797b644c8/lisp/emacs-lisp/package.el#L738) by Emacs
[2]: it does not contain a [foo-pkg.el](https://github.com/emacs-mirror/emacs/blob/310ed338c195c755b11e0c62bde9629797b644c8/lisp/emacs-lisp/package.el#L717) file

Reply:
Resolved it with https://github.com/NixOS/nixpkgs/pull/278918

Reply:
@jian-lin https://github.com/NixOS/nixpkgs/issues/278925